src/projection.rs

```diff
@@ -132,12 +127,6 @@ impl<S: BaseFloat> From<PerspectiveFov<S>> for Matrix4<S> {
             "The far plane distance cannot be below zero, found: {:?}",
             persp.far
         );
-        assert!(
-            persp.far > persp.near,
```

Comment:
we should probably still assert if they are too close, or otherwise `        let c2r2 = (persp.far + persp.near) / (persp.near - persp.far);` would be screwed up